source/commands/utils/sharedDangerfileArgs.ts

```diff
@@ -23,6 +23,8 @@ export interface SharedCLI extends program.CommanderStatic {
   process?: string
   /** Use a URL to send in the DSL instead of just STDIN */
   passURLForDSL?: boolean
+  /** Fail if there are fails in the danger report */
+  strict: boolean
```

Comment:
Sounds good, but can this please be consistent with the Ruby version and be called `failOnErrors`

https://github.com/danger/danger/blob/13060c145f03fc3f29e2d1f15d156945c5aa7197/lib/danger/commands/runner.rb#L62